src/script/auth/page/OAuthPermissions.tsx

```diff
@@ -0,0 +1,299 @@
+              <Box
+                style={{
+                  marginBottom: '24px',
+                  background: COLOR_V2.GRAY_20,
+                  borderColor: COLOR_V2.GRAY_20,
+                  padding: '8px',
+                }}
+              >
+                <ul
+                  style={{
+                    marginTop: 0,
+                    paddingInlineStart: '20px',
+                    fontSize: '12px',
+                  }}
+                >
+                  {oAuthScope.map((scope, index) => (
+                    <li key={index} style={{textAlign: 'start'}}>
+                      <Text>{_(oauthStrings[scope])}</Text>
+                    </li>
+                  ))}
+                </ul>
+                <Text
+                  data-uie-name="oauth-learn-more"
+                  style={{
+                    fontSize: '12px',
+                    lineHeight: '16px',
+                    display: 'block',
+                  }}
+                >
+                  <FormattedMessage
+                    {...oauthStrings.learnMore}
+                    values={{
+                      // eslint-disable-next-line react/display-name
+                      learnMore: ((...chunks: string[] | React.ReactNode[]) => (
+                        <a
+                          target="_blank"
+                          rel="noopener noreferrer"
+                          data-uie-name="go-learn-more"
+                          href="https://support.wire.com/hc/en-us/articles/9628065287965-Calendar-integration"
+                        >
+                          {chunks}
+                        </a>
+                      )) as any,
+                    }}
+                  />
+                </Text>
+              </Box>
+            )}
+            <Text muted style={{fontSize: '12px', lineHeight: '16px', display: 'block'}} data-uie-name="oauth-details">
+              {_(oauthStrings.details)}
+            </Text>
+            <div style={{display: 'flex', flexDirection: 'row', alignItems: 'center', marginTop: '74px', gap: '16px'}}>
+              <Button
+                variant={ButtonVariant.SECONDARY}
+                style={{margin: 'auto', width: 200}}
+                type="button"
+                onClick={onCancel}
+                data-uie-name="do-oauth-cancel"
+                onKeyDown={(event: React.KeyboardEvent) => {
+                  if (event.key === KEY.ESC) {
+                    onCancel();
+                  }
+                }}
+              >
+                {_(oauthStrings.cancel)}
+              </Button>
+              <Button
+                style={{margin: 'auto', width: 200}}
+                type="button"
+                onClick={onContinue}
+                data-uie-name="do-oauth-allow"
+                onKeyDown={(event: React.KeyboardEvent) => {
+                  if (event.key === KEY.ENTER) {
+                    onContinue();
+                  }
```

Comment:
I think we can use `handleKeyDown` from `KeyboardUtil` here (unless we dont want it to react on space key also)

Reply:
don